sdk/python/kfp/v2/dsl/__init__.py

```diff
@@ -13,6 +13,19 @@
 # limitations under the License.
 
 from kfp.v2.dsl.component_decorator import component
+from kfp.dsl.io_types import (
```

Comment:
I can separate this into a different PR if that's preferable.

Reply:
That's fine.